### PR TITLE
Add Jyothis Thomas to Contributors list

### PR DESCRIPTION
**Pull request by jyothis-thomas**

### CODE REVIEW

File: Contributors.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ﻿# Contributors
 Amal Joseph
+Jyothis Thomas
```
Comment (jyothis-thomas) [inline Contributors.md:3]: I have no idea what this is doing!!!

**Comment by jyothis-thomas** [comment]
Hello Just testing what happens if I reopen pull request.

**Comment by abhaikollara** [comment]
@jyothis-thomas Why was this closed ?

**Comment by abhaikollara** [comment]
Merging it now

**Pull request merged**